src/Spice86.Shared/Interfaces/IGui.cs

```diff
@@ -126,4 +119,9 @@ public interface IGui {
     /// Height of the primary video buffer from the emulator's point of view, in pixels.
     /// </summary>
     int Height { get; }
+
+    /// <summary>
+    /// Refresh the display with the content of the video ram.
+    /// </summary>
+    void UpdateScreen();
```

Comment:
Does this work with additionnal video buffers with custom memory ranges added via GDB ?

Reply:
Not yet. Because just the memory range is not enough. You also need to specify which presenter should be used with it.